src/views/virtualmachines/details/tabs/configuration/storage/components/tables/disk/DiskRow.tsx

```diff
@@ -60,6 +69,15 @@ const DiskRow: FC<
 
   const hasPVC = isPVCSource(obj);
 
+  const { displayName } = useMemo(() => {
+    const disks = getDisks(vm) || [];
+    const foundDisk = disks.find((disk) => disk.name === name);
+    const cdrom = foundDisk && isCDROMDisk(foundDisk);
+    return {
+      displayName: cdrom ? t(CDROM_DEVICE_NAME) : name,
```

Comment:
when using t func on const, it has some problem with the translation, there's 2 things can be done here:
1. add the t func to the constant value (so no need in wrapping with t the const).
2. you can do something like this: `t('{{some_dynamic_constant}}, { some_dynamic_constant: CDROM_DEVICE_NAME });`